### ISSUE TITLE
In Markdown, Unable to Accept Suggestion of GitHub Copilot

### ISSUE THREAD

**Issue by ktomari** [rstudio/rstudio#14129]
I think this counts as a bug, and I don't think I've seen it elsewhere in issues. My apologies if this is known.

### System details

R version 4.3.2 (2023-10-31)
rstudio: 2023.12.0,369
Platform: aarch64-apple-darwin23.0.0 (64-bit)
Running under: macOS Sonoma 14.2.1

### Steps to reproduce the problem

Set up RStudio with GitHub Copilot. Open a qmd file and begin typing in markdown area.

### Describe the problem in detail

I noticed that while typing in the markdown section of a qmd that copilot was offering useful suggestions, but that hitting 'Tab' yielded a regular indentation, and not an acceptance of the completion. I looked in the [docs](https://docs.posit.co/ide/user/ide/guide/tools/copilot.html) for "markdown" and didn't see it listed as a scenario. I tried different keyboard combinations to accept copilot's suggestion, but nothing worked. ChatGPT suggested I try cmd + space, but on macOS that just opens spotlight.

<img width="1003" alt="Screenshot 2024-01-15 at 12 37 07 PM" src="https://github.com/rstudio/rstudio/assets/40505955/ba3b1f47-4fe5-4c03-b9ca-405f1c0e5afc">

### Describe the behavior you expected

I expected it would autocomplete on hitting "Tab", but really that isn't ideal either.

- [X] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [X] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [NA] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.posit.co/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by ronblum** [comment]
@ktomari Thank you for raising this. It sounds like a bug, although I haven't been able to reproduce it. I'll mark it for review, though. Do you see the same thing with R Markdown and R files?

**Comment by ktomari** [comment]
Well, it seems to have been a weird idiosyncrasy. I can't seem to reproduce it now either! I'll close this with this comment.

I'll describe my attempt to reproduce it below as a sort of post-mortem report.

I came back to this issue this morning and so I opened RStudio anew to see if I could reproduce the problem with Rmd files. I couldn't. I tried reproducing it where I had first encountered it (in a qmd while working with devtools to write a package) but I couldn't. Maybe it was just a momentary bug that was cleared after restarting my machine?
